src/main/java/net/imagej/ops/topology/eulerCharacteristic/EulerCharacteristic26NFloating.java

```diff
@@ -32,13 +33,13 @@
  * Bone 14: 173-182.
  * <a href="http://dx.doi.org/10.1016/8756-3282(93)90245-6">doi:10.1016/8756-3282(93)90245-6</a>
  *
- * @author Richard Domander
- * @author Michael Doube
+ * @author Richard Domander (Royal Veterinary College, London)
```

Comment:
BTW, I really don't care if you include your affiliation or not. As you probably noticed, all the U. Konstanz team does, but the UW-Madison team does not. As long as it is consistent for each organization I am fine with whatever.